cfme/tests/configure/test_ntp_server.py

```diff
@@ -98,7 +98,7 @@ def test_ntp_conf_file_update_check(request, appliance, empty_ntp_dict, ntp_serv
 
 @pytest.mark.tier(3)
 def test_ntp_server_check(request, appliance, ntp_servers_keys):
-    request.addfinalizer(partial(clear_ntp_settings, appliance))
+    request.addfinalizer(lambda: appliance.server.settings.update_ntp_servers(empty_ntp_dict))
```

Comment:
@izapolsk has a fix for the exception this causes.

Reply:
yep